### PR TITLE
[MMI] Applied code fencing in transactions and pending tx tracker controllers

### PR DESCRIPTION
**Pull request by albertolive**
## Explanation

Applied code fencing in transactions and pending tx tracker controllers

https://consensyssoftware.atlassian.net/browse/MMI-2495

## Pre-merge author checklist

- [X] I've clearly explained:
  - [X] What problem this PR is solving
  - [X] How this problem was solved

## Pre-merge reviewer checklist

- [X] Manual testing (e.g. pull and build branch, run in browser, test code being changed)
- [X] PR is linked to the appropriate GitHub issue

### CODE REVIEW

File: app/scripts/controllers/transactions/pending-tx-tracker.js [hunk 1]
```diff
@@ -180,7 +187,15 @@ export default class PendingTransactionTracker extends EventEmitter {
 
     // extra check in case there was an uncaught error during the
     // signature and submission process
-    if (!txHash) {
+
+    let hasNoHash = !txHash;
+
+    ///: BEGIN:ONLY_INCLUDE_IN(mmi)
+    // Don't emit noTxHashErr for custodian transactions https://consensys.slack.com/archives/C02612BF2LW/p1664360358834839?thread_ts=1663693387.576109&cid=C02612BF2LW
```
Comment (zone-live) [inline app/scripts/controllers/transactions/pending-tx-tracker.js:194]: @albertolive delete this url from the comment, it creates too much noise

**Review by zone-live — CHANGES_REQUESTED** [review]

**Review by zone-live — DISMISSED** [review]

**Review by shane-t — DISMISSED** [review]

**Comment by albertolive** [comment]
@darkwing could you approve it? I already added more tests

File: app/scripts/controllers/transactions/index.js [hunk 2]
```diff
@@ -1353,6 +1357,15 @@ export default class TransactionController extends EventEmitter {
     // So that we do not increment nonce + resubmit something
     // that is already being incremented & signed.
     const txMeta = this.txStateManager.getTransaction(txId);
+
+    ///: BEGIN:ONLY_INCLUDE_IN(mmi)
```
Comment (albertolive) [inline app/scripts/controllers/transactions/index.js:1361]: MMI does not broadcast transactions, as that is the responsibility of the custodian

**Review by zone-live — DISMISSED** [review]

**Comment by albertolive** [comment]
@zone-live @brad-decker could you review this one?

**Review by pedronfigueiredo — APPROVED** [review]

**Review by zone-live — APPROVED** [review]

**Pull request merged**